software/SchemaTerms/sdocollaborators.py

```diff
@@ -140,23 +142,24 @@ def createCollaborator(cls, file_path):
                 desc = file_handle.read()
             return cls(ref, desc=desc)
         except OSError as e:
-            log.error("Error loading colaborator source %s: %s" % (file_path, e))
+            log.error(f"Error loading colaborator source: {e}")
             return None
 
     @classmethod
     def loadCollaborators(cls):
         if not len(cls.COLLABORATORS):
             for file_path in glob.glob("data/collab/*.md"):
                 cls.createCollaborator(file_path)
-            log.info("Loaded %s collaborators" % len(cls.COLLABORATORS))
+            num_collaborators = len(cls.COLLABORATORS)
+            log.info(f"Loaded {num_collaborators} collaborators")
```

Comment:
I think you can put entire expressions in the {}, so that it could be 

`log.info(f"Loaded {len(cls.COLLABORATORS)} collaborators")`